apps/ui/src/components/pages/dashboard/ApplicationStatusBar.tsx

```diff
@@ -62,18 +60,26 @@ const ApplicationStatusBar = ({ expiresAt }: ApplicationStatusBarProps) => {
 							align="center"
 							gap={20}
 						>
-							{expiresAt ? (
+							{!expiresAt ? (
 								<>
 									<CheckCircleFilled style={{ color: token.colorPrimary, fontSize: 30 }} />
 									<Flex vertical>
-										<Text strong>You have access to PCGL Controlled Data</Text>
-										<Text>Expires: {formatDate(expiresAt)}</Text>
+										<Text strong>{t('dashboard.hasAccess')}</Text>
+										<Text>
+											{t('label.expires')}:
+											{t('date.intlDateTime', {
```

Comment:
<img width="556" alt="image" src="https://github.com/user-attachments/assets/07e85bf9-c7e7-4431-8e1c-05964c11b771" />

Minor, but we should probs add a space between expires and the date